### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by Mikkoun**
- Centered superfluid logo in header.
- Removed a lot of trailing delimiters that looked bad.
- Fixed broken list item grid at the start of most of the pages.
- Replaced token text with the chip all over the project.
- Fixed NextJS scroll bug (now scrolling top when the page changes).
- Disabled filter tooltip autofocus when closing the filter with enter.
- Index page table default height fix.
- Moved copy page link next to breadcrumbs.
- Fixed page layouts/paddings.
- Fixed table loader overflowing corners.
- Changed page header buttons.

### CODE REVIEW

**Comment by elvijsTDL** [comment]
LGTM 👍

**Review by elvijsTDL — APPROVED** [review]

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -101,7 +117,7 @@ const Layout: FC = ({ children }) => {
         // Prefer to show blank page here instead of a loader (less flickering).
         <Box
           sx={{
-            height: "100vh",
+            // height: "100vh",
```
Comment (kasparkallas) [inline src/pages/_app.tsx:120]: zombie

Comment (Mikkoun) [reply src/pages/_app.tsx]: Removed it ;)

**Review by kasparkallas — APPROVED** [review]
Love it!

**Pull request merged**